migrator/migrations/registry.go

```diff
@@ -14,7 +14,7 @@ var (
 
 // MustRegisterMigration registers a Migration, panic-ing if there's an error.
 func MustRegisterMigration(m types.Migration) {
-	if !env.PostgresDatastoreEnabled.BooleanSetting() && m.StartingSeqNum > migrations.CurrentDBVersionSeqNumWithoutPostgres() {
+	if !env.PostgresDatastoreEnabled.BooleanSetting() && m.StartingSeqNum > migrations.BasePostgresDBVersionSeqNum() {
```

Comment:
Does this mean non-postgres migrations beyond `LastRocksDBVersionSeqNum` will be silently ignored ?
If having extra non-postgres migrations is an error, wouldn't it be better to panic here, as suggested in the function documentation comment ?